galette/lib/Galette/Core/Authentication.php

```diff
@@ -292,4 +298,26 @@ public function __set($name, $value)
     {
         $this->$name = $value;
     }
+
+    /**
+     * get user access level
+     *
+     * @return integer
+     */
+    public function getAccessLevel()
+    {
+
+        if ($this->isSuperAdmin()) {
+            return self::ACCESS_SUPERADMIN;
+        } elseif ($this->isAdmin()) {
+            return self::ACCESS_ADMIN;
+        } elseif ($this->isStaff()) {
+            return self::ACCESS_STAFF;
+        } elseif ($this->isGroupManager()) {
+            return self::ACCESS_MANAGER;
+        } else {
```

Comment:
Not logged user should probably be added as well, since self-subscription may use it, no?